pkg/util/strings.go

```diff
@@ -36,6 +36,16 @@ func SplitString(str string) []string {
 	return regexp.MustCompile("[, ]+").Split(str, -1)
 }
 
+// TrimQuotes returns a copy without the surrounding quote marks
+func TrimQuotes(s string) string {
+	if len(s) >= 2 {
```

Comment:
strings.Trim , or  strings.TrimPrefix and strings.TrimSuffix depending on if you care about nested quotes: https://play.golang.org/p/r3Gr_f9G2xn

Reply:
👍